doc/man7/EVP_RAND-HASH-DRBG.pod

```diff
@@ -54,6 +54,23 @@ These parameters work as described in L<EVP_RAND(3)/PARAMETERS>.
 
 =head1 NOTES
 
+When using the FIPS provider, only these digests are permitted (as per
+L<FIPS 140-3 IG D.R|https://csrc.nist.gov/CSRC/media/Projects/cryptographic-module-validation-program/documents/fips%20140-3/FIPS%20140-3%20IG.pdf>):
```

Comment:
This (and below in the HISTORY section) sounds like this hardcoded to enforce the restriction by default, but I think this should be worded differently, mentioning the option.